scripts/build/utils.js

```diff
@@ -0,0 +1,98 @@
+const path = require('path')
+const del = require('del')
```

Comment:
seems like both `rimraf` and `del` are used across the codebase - not a big deal, but could be unified

Reply:
I think the usages of rimraf were only in npm scripts and I removed those ones so I don't think rimraf is being used anymore.